make/STM32F4xx/common.mk

```diff
@@ -95,6 +95,10 @@ ifeq (STM32F446xC, $(DEVICE))
     MAPPED_DEVICE = STM32F446xE
 endif
 
+ifeq (STM32F446RE, $(DEVICE))
```

Comment:
Replace the R with an x to make this work with the device naming convention. See first comment.